config/pimcore/ruleset.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<ruleset name="Youwe/Pimcore">
+    <description>Youwe coding standard extensions for sniffing code in Pimcore projects.</description>
+
+    <!-- Use default Youwe coding standards for PHP projects. -->
+    <rule ref="../default" />
+
+    <!-- Pimcore specific exclusions -->
+    <exclude-pattern>*/bin/*</exclude-pattern>
+    <exclude-pattern>*/install-profiles/*</exclude-pattern>
+    <exclude-pattern>*/public/*</exclude-pattern>
+    <exclude-pattern>*/var/*</exclude-pattern>
```

Comment:
Are these files relative to the project root? 

If so, why do these patterns start with `*/` intsead of only excluding the `/project-root/var/` folder?
If not, then should we apply the same trick as you did with phpcs to avoid `/var/www/html` to be excluded?